lib/grape/cookies.rb

```diff
@@ -32,7 +32,7 @@ def each(&block)
     end
 
     def delete(name, opts = {})
-      options = opts.merge(value: 'deleted', expires: Time.at(0))
+      options = opts.merge!(value: 'deleted', expires: Time.at(0))
```

Comment:
Why? Both `options` and `opts` are local variables, they aren't used somewhere else except there.